packages/visx-text/test/Text.test.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
-import { shallow, mount } from 'enzyme';
+import { shallow } from 'enzyme';
```

Comment:
I can't comment on it since it wasn't changed, but wonder if we should just remove the two `shallow` tests here since they only assert that there are not errors. then we'd be fully on RTL as well 👍

Reply:
yea sure, these tests are trivial. For later iterations, I would like to convert the shallow tests to RTL as well (these are relatively easier, most of them are just renaming work), so we can fully remove enzyme out of a devDependencies.